### PR TITLE
Add readme file to each submodule

### PR DESCRIPTION
**Pull request by jakcinmarina**

### CODE REVIEW

File: packages/tatum-one/README.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+|                       | **AuctionCancel**               | Cancel auction on the auction. Only possible for the seller or the operator.                                          |
+|                       | **AuctionSettle**               | Settle auction. There must be buyer present for that auction.                                                         |
+|                       | **AuctionUpdateFee**            | Update auction fee.                                                                                                   |
+|                       | **AuctionUpdateFeeRecipient**   | Update auction fee recipient.                                                                                         |
+|                       | **AuctionApproveNftTransfer**   | Approve NFT transfer for auction to perform listing of the asset.                                                     |
+|                       | **AuctionApproveErc20Transfer** | Approve ERC20 transfer for auction to perform bidding on the asset in the auction.                                    |
+
+#### MarketPlace
```
Comment (ruckim) [inline packages/tatum-one/README.md:79]: Please modify MarketPlace -> Marketplace everywhere.

File: packages/tatum-algo/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+#### Blockchain
+
+| **Category**   | Function                        | Description                                           |
+| -------------- | ------------------------------- | ----------------------------------------------------- |
+| _Account_      | **getBlockchainAccountBalance** | Returns account balances for a given address.         |
+| _Block_        | **getBlock**                    | Get blockchain block by hash or block number.         |
+|                | **getCurrentBlock**             | Returns current block.                                |
+| _Transactions_ | **getTransaction**              | Returns transaction by hash from algorand blockchain. |
+|                | **getTransactionsCount**        | Returns count of transactions.                        |
+| _Broadcast_    | **postBroadcast**               | Returns information about Ada blockchain.             |
```
Comment (ruckim) [inline packages/tatum-algo/README.md:12]: Ada -> Algo

File: packages/tatum-bch/README.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+#### Blockchain
+
+| **Category**   | Function            | Description                                           |
+| -------------- | ------------------- | ----------------------------------------------------- |
+| _Block_        | **getBlock**        | Get blockchain block by hash or block number.         |
+|                | **getCurrentBlock** | Returns current block.                                |
+|                | **getBlockHash**    | Returns transactions for address from Bch blockchain. |
+| _Transactions_ | **getTransaction**  | Returns transaction by hash from Bch blockchain.      |
+|                | **getTxForAccount** | Returns count of transactions.                        |
+| _Broadcast_    | **postBroadcast**   | Returns information about Ada blockchain.             |
```
Comment (ruckim) [inline packages/tatum-bch/README.md:12]: Ada -> Bch

File: packages/tatum-bnb/README.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+## Binance chain (BNB) - List of features
+
+#### Blockchain
+
+| **Category** | Function          | Description                                     |
+| ------------ | ----------------- | ----------------------------------------------- |
+| _Account_    | **getAccount**    | Returns account by address from Bnb blockchain. |
+| _Broadcast_  | **postBroadcast** | Returns information about Ada blockchain.       |
```
Comment (ruckim) [inline packages/tatum-bnb/README.md:8]: Ada -> Bch

File: packages/tatum-bsc/README.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+| _Block_        | **getBlockchainAccountBalance** | Returns balance on address from Bsc blockchain.                                    |
+| _Account_      | **getAccount**                  | Returns account by address from Bsc blockchain.                                    |
+|                | **getAccountBep20Address**      | Returns account BEP20 address from Bsc blockchain.                                 |
+|                | **getBlock**                    | Returns block by its hash from Bsc blockchain.                                     |
+| _Transactions_ | **getTransactionsCount**        | Returns a number of outgoing Bsc transactions for the address from Bsc blockchain. |
+|                | **getTransaction**              | Returns transaction by hash from Bsc blockchain.                                   |
+|                | **estimateGas**                 | Returns gasLimit and gasPrice estimation of the transaction from Bsc blockchain.   |
+| _Broadcast_    | **postBroadcast**               | Returns information about Ada blockchain.                                          |
```
Comment (ruckim) [inline packages/tatum-bsc/README.md:14]: Ada -> Bsc

File: packages/tatum-btc/README.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
+## Bitcoin (BTC) - List of features
+
+#### Blockchain
+
+| **Category**   | Function            | Description                                                                 |
+| -------------- | ------------------- | --------------------------------------------------------------------------- |
+| _Block_        | **getBlock**        | Get blockchain block by hash or block number.                               |
+|                | **getCurrentBlock** | Returns information about Btc blockchain.                                   |
+|                | **getBlockHash**    | Returns transactions for address from Bch blockchain.                       |
```
Comment (ruckim) [inline packages/tatum-btc/README.md:9]: Btc

File: packages/tatum-btc/README.md [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+|                | **getBalance**      | Returns balance on address from Btc blockchain.                             |
+| _Transactions_ | **getTransaction**  | Returns transaction by hash from Bch blockchain.                            |
```
Comment (ruckim) [inline packages/tatum-btc/README.md:11]: Bch -> Btc

**Review by ruckim — APPROVED** [review]

**Comment by Hathoriel** [comment]
I like that you also changed the templates 👍

**Pull request merged**